assets/css/index.sass

```diff
@@ -47,6 +47,31 @@ $bottom-bar-height: 25px
     .modal-close:hover
         background: #909090
 
+#settings
+    .modal-contents
+        margin: 10% auto
+        width: 400px
+        padding: 20px
+        border-radius: 10px
+
+#import-file
+    .modal-contents
+        margin: 10% auto
+        width: 400px
+        padding: 20px
+        border-radius: 10px
```

Comment:
same here.  it may be reasonable to re-specify, but i'm fine with using defaults